src/pages/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { useContext, useState } from "react";
+import useDbData from "./useDbData";
+import { MainPageWrapper, AddTodoBtn, TodosWrapper } from "./MainPage.styles";
+import Todo from "../../components/Todo/Todo";
+import AddTodo from "../../components/AddTodo";
+import { DarkModeContext } from "../../contexts/DarkModeContext";
+
+interface MainPageProps {
+  search: string;
+}
+export default function MainPage({ search }: MainPageProps) {
+  const [addTodoActive, setAddTodoActive] = useState(false);
+  const todos = useDbData();
+  const { isDarkMode } = useContext(DarkModeContext);
+  return (
+    <MainPageWrapper isDarkMode={isDarkMode}>
+      <AddTodoBtn
+        onClick={() => setAddTodoActive(true)}
+        isDarkMode={isDarkMode}
+      >
+        +
+      </AddTodoBtn>
+      <TodosWrapper>
+        {todos
+          ?.filter((todo) => {
+            return (
+              todo.name.toLowerCase().startsWith(search.toLowerCase()) ||
```

Comment:
to w sumie możesz wyrzucić do osobnej funkcji - mam na myśli to, co znajduje się w filter

`(todo)=> todo.name.toLowerCase() .........`

Reply:
ale też nie jest to wymagane, na pewno warte rozważenia gdy mamy bardziej złożone callbacki